apps/forms/models.py

```diff
@@ -21,8 +23,23 @@ class FormField(AbstractFormField):
                        related_name='form_fields')
 
 
+class WagtailCaptchaFormBuilder(FormBuilder):
+    @property
+    def formfields(self):
+        # Add captcha to formfields property
+        fields = super().formfields
+        fields['captcha'] = CaptcheckCaptchaField(
+            label=_('I am not a robot'),
+            help_text=_('If you are having difficulty please contact us, '
+                        'details adjacent'))
```

Comment:
I think the details adjacent is true for a+, but not here?

Reply:
yes, and we also need to remove that anyway as it doesn't pass the bitv test

Reply:
todo: need wording here